pygmt/src/triangulate.py

```diff
@@ -89,13 +89,12 @@ def _triangulate(
         {projection}
         {region}
         {spacing}
-        outgrid : bool or str
-            The name of the output netCDF file with extension .nc to store the
-            grid in. The interpolation is performed in the original
-            coordinates, so if your triangles are close to the poles you are
-            better off projecting all data to a local coordinate system before
-            using ``triangulate`` (this is true of all gridding routines) or
-            instead select :gmt-docs:`sphtriangulate <sphtriangulate.html>`.
+        {outgrid}
+            The interpolation is performed in the original coordinates, so if
+            your triangles are close to the poles you are better off projecting
+            all data to a local coordinate system before using ``triangulate``
+            (this is true of all gridding routines) or instead select
+            :gmt-docs:`sphtriangulate <sphtriangulate.html>`.
```

Comment:
We also need to update 
https://github.com/GenericMappingTools/pygmt/blob/c0921af6568c7c48f2671b83a81685fc722916b9/pygmt/src/triangulate.py#L206-L212

Reply:
Thanks for catching it. Fixed in c439204d675010d4d3a59e642e115af16f0da578.